src/addItem.js

```diff
@@ -0,0 +1,19 @@
+import * as TaskManager from './taskManager.js';
+import { createItems } from './createItems.js';
+
+function addTask() {
+  const inputField = document.getElementById('add-tasks');
+  const description = inputField.value;
+
+  if (description.trim() === '') {
+    return;
+  }
+  TaskManager.addTask(description);
+  TaskManager.saveTasks();
+
+  inputField.value = '';
+
+  createItems();
+}
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] As i have suggested above, you are not meant to use any linters diable rule in your codebase. The linters are there to make sure your code meets the best standard practices. I'll suggest you remove this linter diable rule and fix the issue. :+1: :+1: :+1: